src/commands/service/create.ts

```diff
@@ -10,10 +10,6 @@ import {
   YAML_NAME as BedrockFileName,
 } from "../../lib/bedrockYaml";
 import { build as buildCmd, exit as exitCmd } from "../../lib/commandBuilder";
-import {
-  PROJECT_CVG_DEPENDENCY_ERROR_MESSAGE,
```

Comment:
If these constants aren't being used anywhere in our project, we can probably remove them

Reply:
@mtarng I have removed them and also `PROJECT_INIT_DEPENDENCY_ERROR_MESSAGE` which is not used. thanks